src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -155949,5 +155949,26 @@ void M()
                 Diagnostic(ErrorCode.WRN_NullabilityMismatchInReturnTypeOfTargetDelegate, "d2").WithArguments("ref readonly string? D<string?>.Invoke()", "D<string>").WithLocation(9, 28)
                 );
         }
+
+        [Fact, WorkItem(59336, "https://github.com/dotnet/roslyn/issues/59336")]
+        public void ParameterNameCrash()
```

Comment:
Consider changing the test name to be more specific. Perhaps `AnonymousMethodNoSignatureAndDelegateAttributes`?